### DISCUSSION TITLE
Help with flagged account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22459] [New to GitHub]
<p>My alternative GitHub account (<a href="https://github.com/forestgeoinformatics" rel="noopener nofollow ugc">https://github.com/forestgeoinformatics</a>) has been flagged. I do not know why it has been flagged. Please help as I want to continue work there.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>try this but without assurance,</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by ghost** [comment]
<p>Thanks. I’ll check. :smiley:</p>

**Reply by nogeek-net** [reply]
hello  , i am a chinese user ,  i account was ben flagged,

i don't know how to fixed it, can you hele me.？？？

(nogeek-net)   this account is very import......

**Comment by friscolt** [comment]
<p>Does GitHub give you a solution?</p>

**Comment by NakatuddeHarrietBrenda** [comment]
Why was ,my account flogged please i need my work😪

**Comment by aliyumuhammedbagana-netizen** [comment]
Hello

**Comment by aliyumuhammedbagana-netizen** [comment]
Aliyumuhammedbagana@gmail.com

**Comment by krylov48** [comment]
My account name is krylov48. I get message "This account is flagged and therefore cannot authorize a third party application' every time plan deploy a Streamlit app. I started to get this message about 2 years ago.

**Comment by daree-dev** [comment]
my account flag pls help

**Comment by mandeepsingh0510** [comment]
whe im trying to login using git hub account its showing this account is flagged, and therefore cannot authorize a third party application any solution guys??
